test/prompt/prompt.go

```diff
@@ -25,11 +25,14 @@ import (
 	"gotest.tools/v3/assert"
 )
 
+// PromptTest provides test utility for prompt test.
 type PromptTest struct {
```

Comment:
Mmm... Should I write this test case for golint errors 🤔??

Reply:
```
I1224 05:03:26.489] test/prompt/prompt.go:29:6: type name will be used as prompt.PromptTest by other packages, and that stutters; consider calling this Test (golint)
I1224 05:03:26.490] type PromptTest struct {
```

So I think it wants you to remove `Test` from the struct name.

Reply:
Ahh... I totally understood, thanks!!
fixed the name :thumbsup: